docs/source/tutorial/spaceflights_tutorial.md

```diff
@@ -1,59 +1,68 @@
-
-In the text, we assume that you create an empty project and follow the flow of the tutorial by copying and pasting the example code into the project as we describe. This tutorial will take approximately two hours, during which time you will learn each step of the Kedro project development workflow, by working on an example to construct nodes and pipelines for the price-prediction model.
+In the text, we assume you have started with an empty Kedro project and we show the steps necessary to convert it into a working project. The tutorial guides you to copy and paste example code into the Kedro project. It takes approximately **one hour** to complete.
 
 ```{note}
-You may prefer to get up and running more swiftly so we provide the full spaceflights example project as a [Kedro starter](../get_started/starters.md). To create the project, run `kedro new --starter=spaceflights`. When prompted for a project name, enter `Kedro Tutorial`. This will generate a project from the [Kedro starter for the spaceflights tutorial](https://github.com/kedro-org/kedro-starters/tree/main/spaceflights) so you can follow the tutorial without any of the copy/pasting.
+You may prefer to get up and running more swiftly. We also provide the example as a [Kedro starter](../get_started/starters.md) you can follow along without copy/pasting.
 ```
 
-## Kedro project development workflow
+## Scenario
 
-When you build a Kedro project, you will typically follow a standard development workflow:
+*It is 2160, and the space tourism industry is booming. Globally, thousands of space shuttle companies take tourists to the Moon and back. You have been able to source data that lists the amenities offered in each space shuttle, customer reviews, and company information.*
 
-![](../meta/images/typical_workflow.png)
+***Project***: *You want to construct a model that predicts the price for each trip to the Moon and the corresponding return flight.*
 
-### 1. Set up the project template
+![](../meta/images/moon-rocket.gif)
 
-* Create a new project with `kedro new`
-* Install project dependencies with `pip install -r src/requirements.txt`
-* Configure the following in the `conf` folder:
-	* Logging
-	* Credentials
-	* Any other sensitive / personal content
+## Get help
+If you hit an issue with the tutorial, the Kedro community can help!
 
-### 2. Set up the data
+Things you can do:
 
-* Add data to the `data/` folder
-* Reference all datasets for the project in the `conf/base/catalog.yml` file
+* check the [Spaceflights tutorial FAQ](spaceflights_tutorial_faqs.md) to see if we have answered the question already
+* search the [archive of Discord discussions](https://linen-discord.kedro.org/)
+* use the [#questions channel](https://kedro-org.slack.com/archives/C03RKP2LW64) on our Slack channel (which replaces our Discord server) to ask the community for help
```

Comment:
This is awesome, I wonder if it would be useful to add that one way of checking it to create a starter and compare it?